### DISCUSSION TITLE
Updating the Elastic Stack

### DISCUSSION THREAD

**Discussion by peasead** [Security-Onion-Solutions/securityonion#4425] [Unsupported Versions] [answered]
Is there a way to update the Elastic Stack outside of the `soup` process? I checked the docs, but I didn't see it...apologies if I missed that.

I'm running `2.3.51`.

**Comment by TOoSmOotH** [comment, marked as answer]
No the Elastic version is tied to the version of SO.

**Reply by defensivedepth** [reply]
@peasead Is there a particular reason you would like to update the elastic stack outside of soup?

**Reply by peasead** [reply]
With new releases of Kibana, there comes new detection logic for the Security App, new integrations for the Elastic Agent, central osquery management, a new architecture for Elastic Fleet, a new row renderer for threat intel matches, etc.

I totally understand version pinning, but those are just a few of the reasons I was asking.

**Comment by TOoSmOotH** [comment]
https://github.com/Security-Onion-Solutions/securityonion/discussions/4434

**Reply by peasead** [reply]
Oh, awesome. How timely!
